op-program/host/config/chaincfg.go

```diff
@@ -0,0 +1,41 @@
+package config
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+var OPGoerliChainConfig = &params.ChainConfig{
```

Comment:
Have defined this here because it's currently only used by the fault proof program.  We possibly should move it into op-geth as the canonical source and it could then be updated there when new hard forks are scheduled.